apps/frontend/components/FormValidation/DroughtFormValidation/DroughtTables.tsx

```diff
@@ -32,20 +25,16 @@ export const DroughtTables = ({
 
   return (
     <>
-      <DisasterTable
-        columns={NumAffectedColumns}
-        columnGroup={NumAffectedColumnGroup}
-        data={[{ id: 1, ...value }]}
-        onChange={onChange}
-        isEditable={isEditMode}
-      />
-      <DisasterTable
-        columns={SocialSectorAffectedColumns(intl)}
-        columnGroup={SocialSectorAffectedColumnGroup}
-        data={[{ id: 1, ...value }]}
-        onChange={onChange}
-        isEditable={isEditMode}
-      />
+      {getDroughtTablesMapping(intl).map((tableSetting, index) => (
+        <DisasterTable
+          columns={tableSetting.columns}
+          columnGroup={tableSetting.columnGroup}
+          data={[{ id: 1, ...value }]}
+          onChange={onChange}
+          isEditable={isEditMode}
+          key={index}
```

Comment:
do we need the key to be a bit more specific? eg. `{drougthValidation-index}` in case multiple disaster tables get shown on one page?